### ISSUE TITLE
Добавить Hide/Show all tracks

### ISSUE THREAD

**Issue by anisart** [wladich/nakarte#572]
Хотелка опциональная, но вроде несложная и интерфейсу не навредит. При планировании дальнейших маршрутов обычно получается вот такая каша
![Screenshot 2020-10-02 122903](https://user-images.githubusercontent.com/693481/94908874-fcf81400-04aa-11eb-9280-e3ad4c9393bf.png)
и, чтобы ее скрыть не удаляя, надо или все прощелкивать, или открывать новую сессию браузера и там все удалить. 

P.S.: И вообще не очень понятно, что происходит, когда удаляешь все треки в соседней вкладке браузера. Открываешь nakarte.me в третьей вкладке, а там снова все треки живы.

**Comment by wladich** [comment]
Про PS - это одно из проявлений бага https://github.com/wladich/nakarte/issues/228

**Comment by wladich** [comment]
Да, начиная с 2 треков надо добавить галочку для выключения всех треков, заодно повесить на какой-нибудь хоткей

Интересный вопрос - сохранять ли при этом состояние для каждого трека? Т.е. должна ли эта галочка менять все галочки для треков или влиять на одтельный статус "показывать видимые треки"?

**Comment by slazav** [comment]
Наверное, либо пункт меню "скрыть/показать", которые выставляют все чекбоксы. Либо внизу кнопка "скрыть/показать", которая чекбоксы не трогает, но все сворачивает и убирает с карты. К этой кнопке можно и кнопки "сохранить треки" (в localStorage), "восстановить" (из localStorage), и м.б. галочка "сохрянять при закрытии вкладки". Сейчас этот самый localStorage совсем непонятно себя ведет (если несколько вкладок), я так к нему и не привык. Либо надо больше контроля, либо - не попользуешься.

С другой стороны, лично для меня это все кажется не очень нужным. Я предпочитаю подгружать с диска то, что надо, когда не надо - все очищать, если хочется сохранить состояние - сохранять ссылку. Так что я пожалуй, за минималистичный вариант, пункт меню "скрыть/показать". Или даже за вариант "ничего не менять":)

**Comment by wladich** [comment]
Fixed in #1357
